.evg.yml

```diff
@@ -261,7 +261,7 @@ buildvariants:
   expansions:
     atlas_server_url: "https://cloud-dev.mongodb.com"
     baas_supportlib_url: "https://s3.amazonaws.com/stitch-artifacts/stitch-support/stitch-support-rhel-70-4.3.2-721-ge791a2e-patch-5e2a6ad2a4cf473ae2e67b09.tgz"
-    go_url: "https://dl.google.com/go/go1.13.6.linux-amd64.tar.gz"
+    go_url: "https://dl.google.com/go/go1.14.12.linux-amd64.tar.gz"
```

Comment:
⚠️ this was prompted by the use of the `log.Lmsgprefix` flag when creating a `logger` in `cli/cli.go` (which allows the timestamp to be printed before log level)

Reply:
baas is on 14.10, we can easily align there and upgrade to 14.12